src/olmo_core/io.py

```diff
@@ -528,6 +528,7 @@ def retriable(
     retriable_errors: Tuple[Type[Exception], ...] = (
         requests.exceptions.ConnectionError,
         requests.exceptions.Timeout,
+        requests.exceptions.ChunkedEncodingError,
```

Comment:
ChunkedEncodingError is not a subclass of ConnectionError but it is retryable and happens a lot with r2